### ISSUE TITLE
The condition of `if` should not be wrapped in parentheses.

### ISSUE THREAD

**Issue by mrapacz** [rrrene/credo#413]
### Environment

* Credo version (`mix credo -v`): 0.8.4
* Erlang/Elixir version (`elixir -v`): 1.4.4
* Operating system: Antergos

### What were you trying to do?
I tried to implement a one line if statement like this:
```
if (is_nil question && answers == []), do: change_question(room, room_id)
```

### Expected outcome
No warnings?

### Actual outcome
I get a Code Reliability issue saying 
```    
The condition of `if` should not be wrapped in parentheses.
```
When I get rid of the parentheses, though, the code doesn't compile saying:
```
** (CompileError) lib/aion/room_channel/room.ex:20: undefined function if/1
``` 
If it's not a bug, I'd be really grateful if you showed me how to do it properly. I couldn't figure how to fix it, though, so in case it actually is a bug, I wanted to report it. Thanks!

**Comment by rrrene** [comment]
Thx for reporting this. :+1:

Since `if` is just a macro and therefore just a function call, the `do:` part is actually a Keyword list passed to the `if` macro call.

```elixir
if(is_nil(question) && answers == [], do: change_question(room, room_id))
```

I'd say this is not a bug in Credo, but I am open to debate that 😄
